x-pack/plugin/sql/src/main/antlr/SqlBase.g4

```diff
@@ -334,7 +339,7 @@ string
 // http://developer.mimer.se/validator/sql-reserved-words.tml
 nonReserved
     : ANALYZE | ANALYZED 
-    | CATALOGS | COLUMNS 
+    | CATALOGS | COLUMNS | CURRENT 
```

Comment:
Why do we need the `CURRENT` alone here?

Reply:
I've discovered it while looking at the grammar for `CURRENT_TIMESTAMP` so it made sense to add it even though we don't have a direct function for it.